Makefile

```diff
@@ -15,6 +16,7 @@ clean-local-properties:
 	rm -f local.properties
 
 sdk: clean-local-properties
+	${EMULATOR} -no-accel -verbose -avd device1 -no-skin -no-audio -no-window&
```

Comment:
This keeps appearing in this PR. This is not necessary to make the `sdk` rule.

Reply:
I guess the best place to put it is in a new rule; something like `make e2e tests` rule...

Reply:
I removing this from this PR since it is extraneous and opening a new PR for make rules for e2e testing